<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>memoryCardClass</class>
 <widget class="QDialog" name="memoryCardClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>387</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>387</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>710</width>
    <height>387</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>memoryCard</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QListWidget" name="icon0list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>341</height>
    </rect>
   </property>
   <property name="iconSize">
    <size>
     <width>144</width>
     <height>80</height>
    </size>
   </property>
   <property name="spacing">
    <number>7</number>
   </property>
   <property name="viewMode">
    <enum>QListView::IconMode</enum>
   </property>
  </widget>
  <widget class="memorycardPic1Label" name="pic1label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>480</width>
     <height>272</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>480</width>
     <height>272</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="memorycardPsfLineEdit" name="gameNameEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Game Name:</string>
   </property>
  </widget>
  <widget class="memoryCardLoadButton" name="LoadButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>EBOOT path:</string>
   </property>
  </widget>
  <widget class="memorycardPsfLineEdit" name="ebootPathEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="gameDirectory">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>memorycardPic1Label</class>
   <extends>QLabel</extends>
   <header>memorycardpic1label.h</header>
   <slots>
    <slot>slot1()</slot>
    <slot>displayPic(QListWidgetItem*)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>memorycardPsfLineEdit</class>
   <extends>QLineEdit</extends>
   <header>memorycardpsflineedit.h</header>
   <slots>
    <slot>setpsfInfo(QListWidgetItem*)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>memoryCardLoadButton</class>
   <extends>QPushButton</extends>
   <header>memorycardloadbutton.h</header>
   <slots>
    <slot>slot1()</slot>
    <slot>enableLoadButton(QListWidgetItem*)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="pcsp.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>icon0list</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>pic1label</receiver>
   <slot>displayPic(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>icon0list</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>gameNameEdit</receiver>
   <slot>setpsfInfo(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>icon0list</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>ebootPathEdit</receiver>
   <slot>setpsfInfo(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelButton</sender>
   <signal>pressed()</signal>
   <receiver>memoryCardClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>icon0list</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>LoadButton</receiver>
   <slot>enableLoadButton(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LoadButton</sender>
   <signal>clicked()</signal>
   <receiver>LoadButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>567</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LoadButton</sender>
   <signal>clicked()</signal>
   <receiver>memoryCardClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gameDirectory</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>memoryCardClass</receiver>
   <slot>gameDirectoryChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>memoryCardClass</sender>
   <signal>accepted()</signal>
   <receiver>memoryCardClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>memoryCardClass</sender>
   <signal>rejected()</signal>
   <receiver>memoryCardClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>gameDirectoryChanged(int)</slot>
  <slot>closeDialog()</slot>
 </slots>
</ui>
